### ISSUE TITLE
[FEATURE] Command Baritone to goto where the FreeCam is

### ISSUE THREAD

**Issue by epifeny** [John200410/rusherhack-issues#1006]
**What feature would you like to be added.**
#TLDR I'm suggesting a feature that would make the player Baritone to where the _FreeCam_ is

Suppose someone has used the _Position_ HUD element (cords). In that case, it would be nice if it were possible to use the mouse left-click to hit either the Overworld X-axis or the Nether X-axis (or they can be only made clickable per the world you're at). It'll be made similar to `*travel` where it would command Baritone to `#goto` these cords. Since these cords change according to _FreeCam_; it would be easy to have the player `#goto` the position of the _FreeCam_ using this feature.

**Is your feature request related to a problem? Please describe.**
No.

**Additional context**
A use case would be, Dungeon stash hunting. You can _FreeCam_ to the dungeon from above ground and command Baritone to `#goto` those cords once you arrive at the dungeon with the _FreeCam_.

**Comment by browsky300** [comment]
#come
